docs/rules/rrd/no-important.md

```diff
@@ -0,0 +1,65 @@
+# No Direct Dom Access
+
+Checks if a component's styles are using the `!important` declaration to enforce styles.
+
+## ❓ Why it's good to follow this rule?
+
+Direct DOM manipulation in Vue components can lead to several issues:
+
+Using `!important` in CSS can lead to several issues:
+
+1. It complicates the cascade of styles, making it harder to understand which styles are being applied.
+2. It increases the specificity of rules, leading to maintenance challenges when updating styles.
+3. It can make debugging more difficult, as the usual cascade and inheritance rules of CSS are bypassed.
+4. It promotes poor coding practices by encouraging developers to rely on `!important` rather than properly managing specificity.
+
+## 😱 Examples of code for which this rule will throw a warning
+
+::: warning
+The following code contains the use of `!important`, which is discouraged:
+:::
+
+
+```vue
+<template>
+  <div class="button">Click Me</div>
+</template>
+
+<style>
```

Comment:
```suggestion
<style scoped>
```

Without the `scope` it will trigger another rule when the analyzer is run within the project itself.

Also. could you use one of the tests as example? It's a good way to keep everything aligned